solr/core/src/test/org/apache/solr/handler/admin/TestCollectionAPIs.java

```diff
@@ -199,13 +199,6 @@ public void testCommands() throws Exception {
     //        "Value of enum must be one of"
     //    );
 
-    compareOutput(
```

Comment:
Because TestCollectionAPIs only works with the "traditional" v2 framework.  This commit moves delete-replica-prop over to the JAX-RS framework, so it can't be covered anymore by TestCollectionAPIs.  You could think of it as the assertion here moving over to the DeleteReplicaPropertyAPITest class added in this PR.